src/MusicalScore/Graphical/GraphicalContinuousDynamicExpression.ts

```diff
@@ -290,13 +290,6 @@ export class GraphicalContinuousDynamicExpression extends AbstractGraphicalExpre
                 this.lines[0].End.x += value;
                 this.lines[1].End.x += value;
             }
-        } else {
-            if (value < 0) {
-                this.lines[0].Start.x += value;
-            } else {
-                this.lines[0].End.x += value;
-                this.lines[1].End.x += value;
-            }
```

Comment:
by the way, this change doesn't do anything. the else block can only be triggered by value = 0 which doesn't do anything.